ansible/inventories/devnet-5/group_vars/all/images.yaml

```diff
@@ -6,7 +6,7 @@ default_ethereum_client_images:
   prysm: ethpandaops/prysm:eip4844-8efb57c
   teku: ethpandaops/teku:4844-interop-297d15c # needs update
   # Execution layer clients
-  besu: ethpandaops/besu:eip-4844-interop-17a4928 # needs update
+  besu: robocopsgonemad/besu:4844-devnet-5
```

Comment:
Could you please add the hash of the latest commit too, so we always know which image is deployed just by looking at it?